src/components/Main/Register/Register.component.jsx

```diff
@@ -1,25 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Form from '../Form/Form.component';
 
 const RegisterForm = props => {
-  const state = {
+  const initRegisterState = {
     firstname: '',
     lastname: '',
     email: '',
     password: '',
     confirmPassword: ''
   };
 
+  const [registerState, setState] = useState(initRegisterState);
+
+  const handleChange = e => {
+    const { id, value } = e.target;
+    setState(prevState => ({
+      ...prevState,
+      [id]: value
+    }));
+  };
+
+  const handleRegister = e => {
+    e.preventDefault();
+    const {
+      firstname,
+      lastname,
+      email,
+      password,
+      confirmPassword
+    } = registerState;
+    if (!firstname || !lastname || !email || !password || !confirmPassword) {
+      alert('Пожалуйста, заполните все поля в форме регистрации');
```

Comment:
Default alerts have limited styling option. What are other options to implement the needed behavior?

Reply:
done, added custom notification popups